src/realm/set.hpp

```diff
@@ -700,6 +716,12 @@ inline std::pair<size_t, bool> Set<T>::insert_null()
 template <class T>
 std::pair<size_t, bool> Set<T>::erase_null()
 {
+    if constexpr (std::is_same_v<Mixed, T>) {
+        auto res = erase(BPlusTree<T>::default_value(this->m_nullable));
+        if (res.second)
```

Comment:
Isn't this redundant? The same is done in erase().

Reply:
yes, to be removed.